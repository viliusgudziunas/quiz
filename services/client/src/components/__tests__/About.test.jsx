import React from "react";
import { shallow } from "enzyme";
import renderer from "react-test-renderer";
import About from "../About";

describe("About component", () => {
  it("should match the snapshot", () => {
    const tree = renderer.create(<About />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  const wrapper = shallow(<About />);
  it("should have a title", () => {
    const element = wrapper.find("h1");
    expect(element.length).toBe(1);
    expect(element.get(0).props.className).toBe("title is-1");
    expect(element.get(0).props.children).toBe("About");
  });

  it("should have a paragraph", () => {
    const element = wrapper.find("p");
    expect(element.length).toBe(1);
    expect(element.get(0).props.children).toBe("Add something relevant here");
  });
});
