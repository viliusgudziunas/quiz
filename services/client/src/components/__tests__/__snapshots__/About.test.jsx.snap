// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component should match the snapshot 1`] = `
<div>
  <h1
    className="title is-1"
  >
    About
  </h1>
  <hr />
  <br />
  <p>
    Add something relevant here
  </p>
</div>
`;
